generate-docs/mkdocs-requirements.txt

```diff
@@ -2,7 +2,6 @@ fontawesome_markdown
 markdown
 mdx_truly_sane_lists
 mdx_unimoji
-mike
```

Comment:
Why did you remove this?

Reply:
Because this action requires the latest mike.

https://github.com/autowarefoundation/autoware-github-actions/pull/4/files#diff-e452dcac3881d99adfa804391ebf04a95794941fa9071c0d0a0c682f4d9f97eeR28-R31

Reply:
It's just a temporary workaround.
If there is no problem, please keep this.